sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/EmbeddingItem.java

```diff
@@ -19,15 +20,37 @@
 @Immutable
 public final class EmbeddingItem implements JsonSerializable<EmbeddingItem> {
 
+    /*
+     * List of embeddings value for the input prompt. These represent a measurement of the
+     * vector-based relatedness of the provided input.
+     */
+    @Generated
+    private final List<Double> embedding;
```

Comment:
Why do we use `List<Double>` instead of `List<Float>` here?

Reply:
It is default to Double when generated.

Reply:
![image](https://github.com/user-attachments/assets/3959d5ca-4281-49ad-bb58-bbfb3074410f)